zazu/util.py

```diff
@@ -44,13 +49,15 @@ def __init__(self, **entries):
 
 
 def check_output(*args, **kwargs):
+    """Like subprocess.check_output but raises an exception if it cannot be found."""
     try:
         return subprocess.check_output(*args, **kwargs)
     except OSError:
         raise_uninstalled(args[0][0])
 
 
 def call(*args, **kwargs):
+    """Like subprocess.call but raise an exception if it cannot be found."""
     try:
         return subprocess.call(*args, **kwargs)
```

Comment:
call() will return the process's return code. check_call would raise a CalledProcessError if this is not zero. Regardless an OSError is raised if the binary can't be found. The goal of this helper is to catch the OSError and tell you how to install the needed command.